### PR TITLE
Style claim and invest page.

### PR DESCRIPTION
**Pull request by fairlighteth**
# Summary

- Styles the invest description page
Screen Shot 2022-01-25 at 14 51 48
Screen Shot 2022-01-25 at 14 51 11
Screen Shot 2022-01-25 at 14 51 00

### CODE REVIEW

File: src/custom/pages/Claim/styled.ts [hunk 1]
```diff
@@ -1179,3 +1179,37 @@ export const CowSpinner = styled.div`
...
+  `}
+  > span {
+    padding: 24px;
+    background: ${({ theme }) => theme.blueShade3};
+    border-radius: 12px;
+    position: relative;
+  }
+
+  > span::before {
+    content: 'Step ' attr(data-step);
```
Comment (W3stside) [inline src/custom/pages/Claim/styled.ts:1201]: i don't personally like this it makes it quite hard to change later on if we forget it's in the CSS. Can't you just write this out in the HTML of `InvestmentFlow`?

**Comment by elena-zh** [comment]
Looks great besides the stepper in a mobile view. 
image

**Comment by W3stside** [comment]
point this to `release/1.10`

**Review by alfetopito — APPROVED** [review]
I'm pretty sure I left a review on this a few moments ago...
Or maybe I did in another PR 🤷 

Anyway, looks good, except the stepper as Elena points out

File: src/custom/pages/Claim/styled.ts (hunk 1 above)
Comment (biocom) [reply src/custom/pages/Claim/styled.ts]: @W3stside Agreed, addressed now.

File: src/custom/pages/Claim/InvestmentFlow/index.tsx [hunk 2]
```diff
@@ -203,25 +204,32 @@ export default function InvestmentFlow({ hasClaims, isAirdropOnly, ...tokenAppro
...
+            <span data-step="Step 2">
+              <p>
+                Claim your vCOW tokens for the Airdrop (available immediately) and for your investment (vesting linearly
+                over 4 years).
+              </p>
+            </span>
+          </StepExplainer>
+          <p>
+            For more details around the token, please read{' '}
+            <ExternalLink href={COW_LINKS.vCowPost}>the blog post</ExternalLink>. For more details about the claiming
```
Comment (anxolin) [inline src/custom/pages/Claim/InvestmentFlow/index.tsx:229]: nit: should we give a line break for the other `For more details...` ?

**Review by anxolin — APPROVED** [review]
Looks super nice other than what has been reported

Not from this PR, but i just noticed while testing this. The vCOW logo shrunk too much:
image

**Comment by biocom** [comment]
@elena-zh Pushed a fix for that:
Screen Shot 2022-01-25 at 19 21 36

**Comment by biocom** [comment]
> Looks super nice other than what has been reported
> 
> Not from this PR, but i just noticed while testing this. The vCOW logo shrunk too much: image

Will review in a separate PR. Might have addressed it already.

**Comment by elena-zh** [comment]
@biocom , the 2nd step text is left-aligned now on all types of mobile devices when screen width is less than 395 px
image

**Comment by biocom** [comment]
@elena-zh should be fixed now.

**Comment by elena-zh** [comment]
Looks good, thanks!

**Pull request merged**